modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -965,6 +964,7 @@ class InternalFFMpegRegister
         /* register a callback function for synchronization */
         av_lockmgr_register(&LockCallBack);
 #endif
+        initLogger_();  // update logger setup unconditionally (GStreamer's libav plugin may override these settings)
```

Comment:
@vrabaud Could you please drop this comment as it is not relevant anymore (al least it doesn't re-set logger settings on each ffmpeg stream)

Reply:
Done